rapidpro/src/main/resources/securityRules.json

```diff
@@ -0,0 +1,19 @@
+[
+  {
+    "active": true,
+    "pattern": "/**/rapidpro/**",
+    "supportedSchemes": [
```

Comment:
You're right, the default settings should be good enough for the time being. I removed the security rules file. If the requirements change to require more granular options for security, I'll add one then.